docs/simulators/OpenFOAM.md

```diff
@@ -126,6 +126,216 @@ task.download_outputs()
+setup, which is part of the [ExaFOAM benchmarks](https://exafoam.eu/benchmarks/).
+Specifically, this case corresponds to the MB9 micro-benchmark, which is
+preparatory work leading up to the HPC Grand Challenge test case of the High
+Lift Common Research Model (CRM-HL). The CRM-HL is a full aircraft configuration
+with deployed high-lift devices, simulated using wall-modeled LES (WMLES). 
+
+The MB9 micro-benchmark captures the essential characteristics of the Grand
+Challenge (such as flow physics and simulation approach) while requiring
+significantly fewer computational resources. It features a two-dimensional,
+three-element high-lift wing configuration, simulated with the IDDES model,
+which provides WMLES functionality in regions of resolved near-wall turbulence.
+This case is based on the well-known 30P30N test case, extensively studied in
+the 4th AIAA CFD High Lift Prediction Workshop (HLPW-4) and supported by
+available experimental data.
+
+You can download the input files for this specific case from the
+[OpenFOAM HPC repository](https://develop.openfoam.com/committees/hpc/-/tree/develop/compressible/rhoPimpleFoam/LES/highLiftConfiguration).
+
+### Prerequisites
+
+Before running the simulation, ensure you have downloaded the input files. You
+can get them from the following
+[repository](https://develop.openfoam.com/committees/hpc/-/tree/develop/compressible/rhoPimpleFoam/LES/highLiftConfiguration).
+Once downloaded, move them into a directory named `highLiftConfiguration` inside
+your working folder.
+
+Here’s an example of how your directory structure should look after downloading
+the necessary files:
+
+```bash
+ls -lasgo highLiftConfiguration
+total 104
+ 0 drwxrwxr-x@  12     384 Jun 13 10:09 .
+ 0 drwx------@ 234    7488 Sep 19 15:15 ..
+ 0 drwxrwxr-x@  11     352 Jun 13 10:09 0.orig
+ 8 -rwxr-xr-x@   1     626 Jun 13 10:09 Allclean
+16 -rwxr-xr-x@   1    6998 Jun 13 10:09 Allrun
+ 8 -rw-rw-r--@   1     991 Jun 13 10:09 COPYING
+48 -rw-rw-r--@   1   21547 Jun 13 10:09 README.md
+ 0 -rw-rw-r--@   1       0 Jun 13 10:09 case.foam
+ 0 drwxrwxr-x@   5     160 Jun 13 10:09 constant
+ 0 drwxrwxr-x@  13     416 Jun 13 10:09 figures
+ 0 drwxrwxr-x@  27     864 Jun 13 10:09 system
+24 -rw-rw-r--@   1   11399 Jun 13 10:09 thumbnail.png
+```
+
+### Configuring and Running the Simulation
+
+We'll run the simulation on a virtual machine with 180 CPUs using the
+`c3d-standard-180` machine type. The simulation involves several preprocessing
+steps, including mesh generation (`blockMesh`, `snappyHexMesh`), and finally,
+the simulation run itself. 
+
+Below is the Python code to configure and run the OpenFOAM simulation:
+
+```python
+import inductiva
+
+machine_group = inductiva.resources.MachineGroup(machine_type="c3d-standard-180")
+machine_group.start()
+
+import inductiva
+
+# Set simulation input directory
+input_dir = "/path/to/highLiftConfiguration"
+
+# Set the simulation commands
+commands = [
```

Comment:
Can't all this be in a file?
I think this is one of UI the issues we need to solve :)
We can read all the lines to a list, right?
Or we can have a parameter called commands_file.
@hpenedones wdyt?

Reply:
Yes, sounds like a good idea!

Reply:
We don't need a parameter for that. We can read a txt file with one command per line. It's pretty simple to read it into a list of strings.
```python
# Read the simulation commands
with open(input_dir + '/input.txt', 'r') as file:
    commands = [line.strip() for line in file]
```

I will update with this.

Reply:
But I wouldn't mind a parameter and do that on the client side.
That way we can hide all this from the user. We want to make thing simpler.
Plus, you code in not correct. You should be using os.path.join :)